packages/fxa-payments-server/src/components/PaymentConfirmation/index.scss

```diff
@@ -9,70 +9,6 @@
 .payment-confirmation {
```

Comment:
Suggestion when we get to editing this, please let me know if you are aligned and I'll update the ticket:
- Remove margin-top in `header` in `PaymentConfirmation` component
- In `checkout-view()`, update from `padding: 0 48px 48px` to `px-12 pb-12`

Reply:
I updated the ticket, thanks!